prod/westeurope/support/functions_backoffice/function_app/terragrunt.hcl

```
@@ -121,6 +121,6 @@ inputs = {
   }
 
   allowed_subnets = []
```

```suggestion
  allowed_subnets = [
    dependency.subnet.outputs.id,
    dependency.subnet_apimapi.outputs.id
  ]
```
Not blocking and not related to this PR but I saw that subnet restrictions are missing.

---

Absolutely, you are right. I have create a different story to improve this point:
https://www.pivotaltracker.com/story/show/176918322